src/erlcloud_ddb2.erl

```diff
@@ -1858,32 +2042,46 @@ update_item(Table, Key, Updates, Opts, Config) ->
 
 -type update_table_return() :: ddb_return(#ddb2_update_table{}, #ddb2_table_description{}).
 
--type global_secondary_index_update() :: {index_name(), pos_integer(), pos_integer()}.
--type global_secondary_index_updates() :: [global_secondary_index_update()].
+-type global_secondary_index_update() :: {index_name(), pos_integer(), pos_integer()} |
+                                         {index_name(), delete} |
```

Comment:
The prefer way may be {create, ...}, {update, ...} and {delete, ...}  
But to keep a consistent style with the remaining exist code, I have chosen the {index_name(), delete} format.